packaging/debian-sid/snapd.install

```diff
@@ -5,6 +5,7 @@ usr/bin/snap-exec /usr/lib/snapd/
 usr/bin/snap-update-ns /usr/lib/snapd/
 usr/bin/snapd /usr/lib/snapd/
 usr/bin/snap-seccomp /usr/lib/snapd/
+usr/bin/snap-recovery /usr/lib/snapd/
```

Comment:
Do we need this on classic systems? I think it should not be packaged.

Reply:
I'm fine with not packaging it, it just means I need to figure out how to not do that on all the distro packaging we do, which is a bit of a headache on its own :/

Reply:
I'm happy to help. I think this should not be in the `snapd` package. It also has its own dependencies which we should be aware of. You ship it and people start to use it ;)

Reply:
This is no longer shipped (except for ubuntu 16.04+ where we need it currently so that its available in core/snapd snap)